web/views/recurring_jobs.erb

```diff
@@ -34,10 +34,10 @@
             </span>
           </td>
           <td class="text-center">
-            <a class="btn btn-warn btn-xs" href="<%= root_path %>recurring-jobs/<%= URI.escape(job.name) %>/enqueue">
+            <a class="btn btn-warn btn-xs" href="<%= root_path %>recurring-jobs/<%= CGI.escape(job.name) %>/enqueue">
```

Comment:
@stefanahman What do you think about using `ERB::Util.url_encode` instead of `CGI.escape`? As far as I know CGI.escape makes more sense to use when do you need escape query params, while `ERB::Util.url_encode` when you need encode a URL 🤔 

This escape isn't required because modern browsers already handle this, they switch the spaces with `%20`, I made a test removing the escape and as you can see below the browser replaced the spaces with `%20`
![image](https://user-images.githubusercontent.com/20196775/118201310-8a321c80-b42d-11eb-8228-6755f32095b8.png)

But since this was added to make sure the url would be encoded correctly, I think we can keep this, but instead of using `CGI.escape` use `ERB::Util.url_encode` to keep the same behavior that it has before and the same behavior the browsers have by default
```irb
irb(main):002:0> URI.escape 'Foo Job'
=> "Foo%20Job"
irb(main):003:0> CGI.escape 'Foo Job'
=> "Foo+Job"
irb(main):004:0> ERB::Util.url_encode 'Foo Job'
=> "Foo%20Job"
```

Reply:
Absolutely! This makes more sense. Fixed in https://github.com/moove-it/sidekiq-scheduler/pull/334/commits/6d697153119d6e322d3568fbebf419aed4c07c75